src/Form.php

```diff
@@ -197,15 +207,25 @@ protected function initLayout()
     }
 
     /**
-     * Setter for field display rules.
+     * @deprecated use Form::setControlDisplayRules - will be removed in dec-2020
+     */
+    public function setFieldsDisplayRules($rules = [])
+    {
+        'trigger_error'('Method is deprecated. Use Form::setControlDisplayRules instead', E_USER_DEPRECATED);
+
+        return $this->setControlDisplayRules(...func_get_args());
+    }
+
+    /**
+     * Setter for control display rules.
      *
      * @param array $rules
      *
      * @return $this
      */
-    public function setFieldsDisplayRules($rules = [])
+    public function setControlDisplayRules($rules = [])
```

Comment:
did you miss `s` after `Control`? (here and for prop name)

later, we should probably move this to control directly, but no to mix this up for now in this PR...

Reply:
Removing "s" was intentional (double plural https://english.stackexchange.com/questions/227378/usage-of-double-plural)

Reply:
I understand, but in computer science it seems wrong to me - we set (multiple) rules fo (multiple) controls

Reply:
The code as we see it is for humans:) I would keep it that way

Reply:
I disagree quite strongly. Here we have a bad design (rule set of rules, rules not set to child controls) and we need to keep the plurality to underline this

Reply:
No arbiter so one has to concede. Changed now. Moving on ...